modROS.lua

```diff
@@ -354,7 +349,7 @@ function ModROS:publish_laser_scan_func()
             -- note the order of the axes here (see earlier comment about FS chirality)
             base_to_laser_z,
             base_to_laser_x,
-            base_to_laser_y,
+            base_to_laser_y + laser_dy * i,
```

Comment:
Doesn't `laser_dy` already contain the total offset?
```suggestion
            base_to_laser_y + laser_dy,
```

Reply:
oops yes!